packages/react/src/component-tests/IcPageHeader/IcPageHeaderTestData.tsx

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable react/jsx-no-bind */
+
+import {
+  IcBreadcrumb,
+  IcBreadcrumbGroup,
+  IcButton,
+  IcChip,
+  IcNavigationItem,
+  IcStep,
+  IcStepper,
+  IcTextField,
+  IcTypography,
+} from "../..";
+/* eslint-disable react/jsx-no-bind */
+import React, { useRef, useState } from "react";
+import { IcPageHeader } from "../..";
+export const ScrollStickyPageHeader = () => {
+  return (
+    <>
+      <div>
+        <div style={{ margin: "10px" }}>
```

Comment:
I think this margin is causing the page header to not show full width
```suggestion
        <div>
```